FlightStreamDeck.Core/Structs.cs

```diff
@@ -1313,5 +1313,299 @@ public enum TOGGLE_EVENT
+        MF_AS3000_TSC_Vertical_TopKnob_Large_INC,
+        MF_AS3000_TSC_Vertical_TopKnob_Large_DEC,
+        MF_AS3000_TSC_Vertical_TopKnob_Small_INC,
+        MF_AS3000_TSC_Vertical_TopKnob_Small_DEC,
+        
+        MF_KAP140_Push_AP,
+        MF_KAP140_Push_HDG,
+        MF_KAP140_Push_NAV,
+        MF_KAP140_Push_APR,
+        MF_KAP140_Push_REV,
+        MF_KAP140_Push_ALT,
+        MF_KAP140_Push_UP,
+        MF_KAP140_Push_DN,
+        MF_KAP140_Long_Push_BARO,
+        MF_KAP140_Push_BARO,
+        MF_KAP140_Push_ARM,
+        MF_KAP140_Knob_Inner_INC,
+        MF_KAP140_Knob_Inner_DEC,
+        MF_KAP140_Knob_Outer_INC,
+        MF_KAP140_Knob_Outer_DEC,
+        
+        MF_oclock_select,
+        MF_oclock_oat,
+        MF_oclock_control_long,
+        MF_oclock_control,
+        
+        MF_AS530_ENT_Push,
+        MF_AS530_MENU_Push,
+        MF_AS530_FPL_Push,
+        MF_AS530_DirectTo_Push,
+        MF_AS530_CLR_Push_Long,
+        MF_AS530_CLR_Push,
+        MF_AS530_MSG_Push,
+        MF_AS530_OBS_Push,
+        MF_AS530_VNAV_Push,
+        MF_AS530_PROC_Push,
+        MF_AS530_COMSWAP_Push,
+        MF_AS530_NAVSWAP_Push,
+        MF_AS530_RNG_Dezoom,
+        MF_AS530_RNG_Zoom,
+        MF_AS530_RightLargeKnob_Right,
+        MF_AS530_RightLargeKnob_Left,
+        MF_AS530_LeftLargeKnob_Right,
+        MF_AS530_LeftLargeKnob_Left,
+        MF_AS530_RightSmallKnob_Right,
+        MF_AS530_RightSmallKnob_Left,
+        MF_AS530_RightSmallKnob_Push,
+        MF_AS530_LeftSmallKnob_Right,
+        MF_AS530_LeftSmallKnob_Left,
+        MF_AS530_LeftSmallKnob_Push,
+        
+        MF_AS430_ENT_Push,
+        MF_AS430_MENU_Push,
+        MF_AS430_FPL_Push,
+        MF_AS430_DirectTo_Push,
+        MF_AS430_CLR_Push_Long,
+        MF_AS430_CLR_Push,
+        MF_AS430_MSG_Push,
+        MF_AS430_OBS_Push,
+        MF_AS430_PROC_Push,
+        MF_AS430_COMSWAP_Push,
+        MF_AS430_NAVSWAP_Push,
+        MF_AS430_RNG_Dezoom,
+        MF_AS430_RNG_Zoom,
+        MF_AS430_RightLargeKnob_Right,
+        MF_AS430_RightLargeKnob_Left,
+        MF_AS430_LeftLargeKnob_Right,
+        MF_AS430_LeftLargeKnob_Left,
+        MF_AS430_RightSmallKnob_Right,
+        MF_AS430_RightSmallKnob_Left,
+        MF_AS430_RightSmallKnob_Push,
+        MF_AS430_LeftSmallKnob_Right,
+        MF_AS430_LeftSmallKnob_Left,
+        MF_AS430_LeftSmallKnob_Push,
+        
+        MF_adf_AntAdf,
+        MF_adf_bfo,
+        MF_adf_frqTransfert,
+        MF_adf_FltEt,
+        MF_adf_SetRst,
+        
+        MF_TransponderIDT,
+        MF_TransponderVFR,
+        MF_TransponderCLR,
+        MF_Transponder0,
+        MF_Transponder1,
+        MF_Transponder2,
+        MF_Transponder3,
+        MF_Transponder4,
+        MF_Transponder5,
+        MF_Transponder6,
+    }
+
+    public static class EventExtensions
+    {
+
+        public static string EventToMobiFlightEventString(this TOGGLE_EVENT input)
```

Comment:
Can we rename this to EventToEventName or EventToSimConnectEvent or something more generic? The function now has only MobiFlight logic, but I don't want to be that specific in function name.

Reply:
updated, resolving